metricbeat/docs/modules/aerospike.asciidoc

```diff
@@ -25,8 +25,6 @@ in <<configuration-metricbeat>>. Here is an example configuration:
 ----
 metricbeat.modules:
 - module: aerospike
-  metricsets: ["namespace"]
-  enabled: false
   period: 10s
```

Comment:
Personally I'd like to see `period: 10s` removed too. And only show this in the reference config.

Reply:
SGTM. @jsoriano WDYT, should we clean up the period too (in case we don't set a different default)? Will make things even shorter.